### PR TITLE
[Policies] "Customer Connectivity Settings" must be re-named to "Exte…

### PR DESCRIPTION
**Pull request by wolfylambova22**
…rnal User Consent Settings"

Fix naming of parameters of CCS

Related issue: [APPLINK-30422](https://adc.luxoft.com/jira/browse/APPLINK-30422)

### CODE REVIEW

**Comment by wolfylambova22** [comment]
@OHerasym @dev-gh @AGaliuzov @VProdanov @orhan-mehmedov @Kozoriz Please review.

**Review by AGaliuzov — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by orhan-mehmedov — APPROVED** [review]

**Review by dev-gh — APPROVED** [review]

File: app/view/WarningView.js [hunk 1]
```diff
@@ -424,7 +424,7 @@ SDL.warningView = Em.ContainerView
             var self = this;
             this._parentView.set(
               'fade', this
-                .checkForCCS3AnimationSupport()
```
Comment (Kozoriz) [inline app/view/WarningView.js:427]: @wolfylambova22 , there is CSS, not "Customer Connectivity Settings ". 
There is mistake, not CCS, must be CSS (Cascading Style Sheets)
Please change to CSS.

File: app/view/WarningView.js [hunk 2]
```diff
@@ -104,7 +104,7 @@ SDL.warningView = Em.ContainerView
            * Check for webkit fillmode animation support Android < 4 version
            * doesnt support webkit animation fillmode
            */
-          checkForCCS3AnimationSupport: function() {
```
Comment (Kozoriz) [inline app/view/WarningView.js:107]: @wolfylambova22 , and there is too..

**Review by Kozoriz — CHANGES_REQUESTED** [review]

**Review by OHerasym — APPROVED** [review]

File: app/view/WarningView.js (hunk 2 above)
Comment (wolfylambova22) [reply app/view/WarningView.js:107]: @Kozoriz Thank you. Fixed in dd9b3142c94eb8faa79193cb50eb95f6147e0d16.

File: app/view/WarningView.js (hunk 1 above)
Comment (wolfylambova22) [reply app/view/WarningView.js:427]: @Kozoriz Fixed in dd9b314.

**Comment by wolfylambova22** [comment]
@Kozoriz Please finish review.

File: app/view/WarningView.js [hunk 3]
```diff
@@ -104,7 +104,7 @@ SDL.warningView = Em.ContainerView
            * Check for webkit fillmode animation support Android < 4 version
            * doesnt support webkit animation fillmode
            */
-          checkForExternalConsent3AnimationSupport: function() {
+          checkForCCS3AnimationSupport: function() {
```
Comment (Kozoriz) [inline app/view/WarningView.js]: @wolfylambova22 , i understand it is not in scope of your task, but can you change CCS to CSS. This is not your mistake (just misstake in code). Can you?

**Review by Kozoriz — APPROVED** [review]

**Pull request merged**